.evergreen/compile-unix.sh

```diff
@@ -1,6 +1,24 @@
-#!/bin/sh
+#!/bin/bash
 set -o errexit  # Exit the script with error if any of the commands fail
 
+# Find PHP binary path for the requested version
```

Comment:
This was moved over so we can run compile for multiple PHP versions on the same box if we wanted to.